analyzers/tests/SonarAnalyzer.Test/Rules/UnusedPrivateMemberTest.Properties.cs

```diff
@@ -136,6 +136,57 @@ public void Method3()
 
     private int Property12 { get; set; } = 42; // FN
 }
+").Verify();
+
+        [TestMethod]
+        public void UnusedPrivateMember_Properties_Accessors() =>
+            builder.AddSnippet(@"
+using System;
+public class PropertyUsages
+{
+    public int AProperty { private get; set; } // Noncompliant {{Remove the unused private getter 'get_AProperty'.}}
+    public int BProperty { get; private set; } // Noncompliant {{Remove the unused private setter 'set_BProperty'.}}
+    public int CProperty { internal get; set; } // Compliant
+    public int DProperty { get; internal set; } // Compliant
+    protected int EProperty { private get; set; } // Noncompliant {{Remove the unused private getter 'get_EProperty'.}}
```

Comment:
This is out of place. Either remove it because it is not necessary to test for other property accessors than `public`, or add more tests (protected property with private setter, internal property, etc.).

Reply:
It was just to have at least a single test with a different access modifier. I don't understand why it should be removed, I guess we can add also protected but it might be superfluous